pkg/controller/performanceprofile/components/tuned/tuned_test.go

```diff
@@ -176,8 +176,83 @@ var _ = Describe("Tuned", func() {
 					channelsRegex := regexp.MustCompile(`\s*channels=combined ` + strconv.Itoa(reserveCPUcount) + `\s*`)
 					Expect(channelsRegex.MatchString(manifest)).To(BeTrue())
 				})
-				It("should set specific devices with reserved CPUs count", func() {
-					netDeviceName := "enp0s4"
+				It("should set by interface name with reserved CPUs count", func() {
+					netDeviceName := "eth*"
+					profile.Spec.Net = &performancev2.Net{
+						UserLevelNetworking: pointer.BoolPtr(true),
+						Devices: []performancev2.Device{
+							{
+								InterfaceName: &netDeviceName,
+							},
+						}}
+					manifest := getTunedManifest(profile)
+					reservedSet, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
```

Comment:
are we sure the Spec.CPU.Reserved is not null?

Reply:
the validation web hook will fail the profile creations if its null

Reply:
great, thanks